build/syntax-highlight.js

```diff
@@ -6,6 +6,7 @@ const lazy = (creator) => {
   let processed = false;
   return () => {
     if (processed) return res;
+    // eslint-disable-next-line no-undef
```

Comment:
Instead of the disable, can you think of a better implementation for this utility function? I think I cobbled it together from some quick googling without much time to think about it. If there's a cleaner way to do it, I'm all ears.